themes.gitconfig

```diff
@@ -536,3 +536,39 @@
     blame-format = "{author:<18} {commit:<6} {timestamp:<15}"
     blame-palette = "#000000" "#343434"
     zero-style = dim
+
+[delta "platypus"]
+    # Author: https://github.com/sarpuser
+    dark = true
+    side-by-side = true
+    hyperlinks = true
```

Comment:
I sort of think themes shouldn't have an opinion about `hyperlinks` -- would you agree? I.e. it should be set outside the theme.

Reply:
That is very fair. I will remove that.